pkg/partmgr/partmgr.go

```diff
@@ -342,8 +342,12 @@ func (p *DBPartition) GetShardingKeys(name string) []string {
 }
 
 // Return the full path to the specified metric item
-func (p *DBPartition) GetMetricPath(name string, hash uint64) string {
-	return fmt.Sprintf("%s%s_%x.%016x", p.path, name, int(hash%uint64(p.GetHashingBuckets())), hash)
+func (p *DBPartition) GetMetricPath(name string, hash uint64, isAggr bool) string {
+	agg := ""
+	if isAggr {
+		agg = "agg/"
```

Comment:
1. It would be nice to have a design page that describe all aspects of pre-aggr feature.
2. Adding new folder to the table hierarchy has impact on the **Delete**/Retention/Rollup/etc. Now, Delete table command should take care of the nested folders and their content.